### DISCUSSION TITLE
Diskussion: Redlining-Gruppen wie WebGIS4

### DISCUSSION THREAD

**Discussion by cmairamhof** [e-netze/webgis-community#28] [Ideas]
Im Zuge unserer Schulungen taucht dieses Thema immer wieder auf (Agrarbehörde, Forstdienst) - deshalb möchte ich das Thema nochmal aufbringen:

Selektion1 --> Sachverhalt1, Selektion2 --> Sachverhalt2 (Teilung, Tausch, Zusammenlegung, Förderung)

Über die Gruppe lässt sich zentral die Farbe steuern und es lässt sich auch die Gruppe leicht entfernen. Größere Selektionen lassen sich in WebGIS5 nachträglich schwer bearbeiten...

![image](https://user-images.githubusercontent.com/79247887/213379682-c9430d69-9281-488c-a232-e06d4e10c81e.png)

**Comment by gis-eni** [comment]
Ich möchte das Redlining nicht noch komplizierter machen, nur weil es ein paar Spezialanwender gibt.

Für mich das wieder eher ein Thema, wo vielleicht "Editing" das bessere Werkzeug ist. In den neuen Editing Tools wird es auch die Möglichkeit des "Feature Transfers" geben. Damit kann man ausgewählte Objekte in ein anderes Thema (Edit Thema) verschieben bzw. kopieren.

Der Vorgang würde hier dann wie folgt aussehen:

* Anwender selektiert Grundstücke
* Klickt in der Tabelle auf "Feature Transfer" Werkzeug
* Bekommt zu Auswahl (parametrierbar), dass Objekte in zB "Freie Flächenthema" kopiert werden können.
* Die Objekte aus dem "Freie Flächenthema" können dann wieder selektiert werden und die Farbe über "Massenattributierung" geändert werden. Das muss natürlich über die Legende im entsprechenden Dienst möglich sein.

* Optional: Kann man über Autovalues des Usernamen beim Transfer mitschreiben und die Flächen dann per Filter nur dem User anzeigen, der sie erstellt hat.
* Optional: Könnte man beim Transfer auch eine "Projektnummer" mitgeben. Auch nach der können man dann Filter, ...

Vorteil:
* Man kommt weg von den "temporären" Redling Objekten, die irgendwann einmal weg sind, weil der Anwender einen Link vergisst, etc
* Mit Editing Lösungen können Workflows besser ungesetzt werden. Beim Redlining kann jeder irgendwie arbeiten und am Ende hat man nur unstrukturierte Daten ohne großen Mehrwert.
* Neben Farbe und Text können für die Objekte je nach Anwendungsfall auch noch andere Attribute gepflegt werden.
* Die Objekte landen in der Datenbank. Weiterverarbeitung und Auswertung werdem einfacher.

Nachteil:
* Die Workflows müssen definiert und parametriert werden
